reference/versions/0.5.0/reference.json

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
Same as with the version bump, I would do this once PR is merged into master and we want to release it.